src/app/common/rpc/use-rpc-stx-broadcast-transaction.ts

```diff
@@ -66,14 +46,16 @@ function useUnsignedStacksTransactionFromRequest(request: TransactionPayload) {
   return tx.result;
 }
 
-export function useRpcStxCallContract() {
-  const { origin, request, requestId, tabId } = useRpcStxCallContractParams();
-  const stacksTransaction = useUnsignedStacksTransactionFromRequest(request);
+export function useRpcStxBroadcastTransaction(method: WalletMethodNames) {
```

Comment:
Not sure if this is a great name, but trying to indicate this is reused with all rpc stx txs that are broadcast by default.